### PR TITLE
Buffs Combustible Lemons

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
tweak: Combustible Lemons have recieved a buff in general. Now they pack the punch that Wave Bronson wanted.
/:cl:

### CODE REVIEW

**Comment by McBawbaggings** [comment]
>90% above potency lemons will gib people.
A FEELING SO COMPLICATED.
I guess why not though. +1

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -132,24 +132,24 @@
...
 			update_mob()
-			explosion(src.loc,-1,-1,2, flame_range = 3)
+			explosion(src.loc,-1,-1,3, flame_range = 4)
 			qdel(src)
 		if(71 to 90)
 			update_mob()
-			explosion(src.loc,-1,-1,2, flame_range = 4)
+			explosion(src.loc,-1,1,3, flame_range = 5)
 			qdel(src)
 		else
```
Comment (JohnGinnane) [inline code/modules/hydroponics/grown/citrus.dm]: Just be aware that if somehow the seed's potency was less than 0 or greater than 90 then you get the biggest explosion, although I don't think you can get a negative seed potency

**Comment by JamieH** [comment]
build this

**Comment by banthebantz** [comment]
I'LL BURN YOUR HOUSE DOWN

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (PopNotes) [inline code/modules/hydroponics/grown/citrus.dm]: As far as I know, it's not possible to get negative potency.

**Pull request merged**